README.md

```diff
@@ -1,4 +1,28 @@
-﻿# relativity-integration-test-helpers
+﻿# Version 7.4 Updates
+
+Starting in Version 7.4.0.1, TestHelpers will use the [Relativity Kepler Framework](https://platform.relativity.com/RelativityOne/Content/Kepler_framework/Kepler_framework.htm) to replace the methods in Relativity.TestHelpers that use DBContext. This was done in order to make these methods compatible with RelativityOne, and creating Kepler Services instead of using DbContext to run SQL queries is Relativity development best practice.
+
+If you are currently using one of the changed methods (e.g. `ArtifactHelpers.Fields.GetFieldArtifactID(yourFieldname, yourDbContext);`), the method will automatically use a Kepler service instead of DbContext, and you will not have to change your tests. Alternatively, you can still force the use of DbContext and not use Kepler by setting the new `app.config` value ForceDbContext to true like so: `<add key="ForceDbContext" value="true" />.` If you wish to remove DbContext from your tests entirely, you can call the corresponding overloaded method: `ArtifactHelpers.Fields.GetFieldArtifactID(yourFieldName, yourWorkspaceId, new Keplerhelper());`
+
+In order to use Kepler services, an empty application `TestHelpers_Kepler_App.rap` will be automatically uploaded to the Application Library of your test environment. `TestHelpersKepler.Interfaces.dll` and `TestHelpersKepler.Services.dll` will also be automatically uploaded to the resource files and linked to the `TestHelpers_Kepler_App.rap`.
+
+List of changed methods:
+
+```csharp
+Relativity.Test.Helpers.ArtifactHelpers.Document.GetDocumentIdentifierFieldColumnName(IDBContext workspaceDbContext, Int32 fieldArtifactTypeID)
+
+Relativity.Test.Helpers.ArtifactHelpers.Document.GetDocumentIdentifierFieldName(IDBContext workspaceDbContext, Int32 fieldArtifactTypeID)
+
+Relativity.Test.Helpers.ArtifactHelpers.Field.GetFieldArtifactID(String fieldname, IDBContext workspaceDbContext)
+
+Relativity.Test.Helpers.ArtifactHelpers.Field.GetFieldCount(IDBContext workspaceDbContext, int fieldArtifactId)
+
+Relativity.Test.Helpers.ArtifactHelpers.Folder.GetFolderName(int folderArtifactId, IDBContext workspaceDbContext)
+
+Relativity.Test.Helpers.TestHelper.GetGuid(int workspaceID, int artifactID)
+```
+
+# relativity-integration-test-helpers

```

Comment:
@jake-martin  maybe this section about updates should go underneath the overview of what Test helpers is